New package: AstroSolve v0.3.0

JuliaRegistrator opened the pull request:
- Registering package: AstroSolve
- Repository: https://github.com/GenAstro/Epicycle
- Created by: @stevenhughes73
- Version: v0.3.0
- Commit: 3a7197c3e55d65622afdc6a5565fd9b461a96c3f
- Reviewed by: @stevenhughes73
- Reference: https://github.com/GenAstro/Epicycle/commit/3a7197c3e55d65622afdc6a5565fd9b461a96c3f#commitcomment-171493474
- Description: An application for space systems with a nod to the giants who came before us and to how far we’ve come. 
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
AstroSolve is part of the Epicycle space mission design and navigation system.
Initial release - trajectory optimization infrastructure and interfaces.

README - https://github.com/GenAstro/Epicycle/blob/main/AstroSolve/README.md
docs - https://genastro.github.io/Epicycle/AstroSolve/dev/

This package is part of a larger monorepo. The original PR removed the deps to avoid sequential registration and three day timers. This registration contains all deps, which only depends on EpicycleBase which is already merged (see here https://github.com/JuliaRegistries/General/pull/142542#event-21019565512)
`````
<!-- END RELEASE NOTES -->

<!-- 80cc8f6b60f354a8c40306cc5f8995f3dc4c048fe8c10e4a013196824d9627c25a6c41ddff8f6f3fd9e74b7ef40046c4ee034dd7032cf982f5928bb9ec8dcb1f44e7789b4efc41bdac3340b07676291ea4d95802080555a4310f2564ed58e8d2a8356cb36b5f02b08c1201358619ea37155c3bfbb8dd53adb5db8e800837b49e9de1ce2ba15e8c253e4699dd0069c8a7048f27a89029625df28f4d433568ace5c57e7162b1af7c5ca890a8ede2eedcc2bb788d9a44eac9295b24c6d98ba6c5fdf2d5e96c386139519f80bafa536c586dc3e1eb4ad5fbd2af2c3a167cce103927 -->

goerz commented:
As in https://github.com/JuliaRegistries/General/pull/142153#issuecomment-3514242154, changing a dependency does not require a new PR. Retriggering the registration will simply updated the existing PR.

[noblock]

JuliaRegistrator committed:
New package: AstroSolve v0.3.0

UUID: d5c6b872-c897-4dc3-b364-70bab4e75788
Repo: https://github.com/GenAstro/Epicycle.git
Tree: a61f602f541df5549ab43e04752f5b3fd8f14df8

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
